x-pack/plugins/security_solution/public/users/pages/details/details_tabs.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+
+import { UsersTableType } from '../../store/model';
+import { useGlobalTime } from '../../../common/containers/use_global_time';
+
+import { UsersDetailsTabsProps } from './types';
+import { type } from './utils';
+
+import { AllUsersQueryTabBody } from '../navigation';
+import { AllUsersQueryProps } from '../navigation/types';
+
+export const UsersDetailsTabs = React.memo<UsersDetailsTabsProps>(
+  ({ docValueFields, filterQuery, indexNames, usersDetailsPagePath }) => {
+    const { from, to, isInitializing, deleteQuery, setQuery } = useGlobalTime();
+
+    const tabProps: AllUsersQueryProps = {
```

Comment:
why do we need to create the object instead of passing the props directly to the component?

Reply:
No reason, fixing it!